packages/serverpod_shared/lib/src/constants.dart

```diff
@@ -0,0 +1,8 @@
+/// Database constants used by the serverpod framework.
+abstract class DatabaseConstants {
+  /// Current version of the database definition.
```

Comment:
Is there really a need to two versions here? It's unlikely we don't want to move these in concert anyway, and it wouldn't do any harm to do so. 

Suggestion: Keep only `databaseDefinitionVersion`.

Reply:
Makes sense! We could always split them later if required.